dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/HttpHeadersInitializer.java

```diff
@@ -39,6 +39,7 @@ public class HttpHeadersInitializer {
         MULTIPART_BOUNDARY;
     public static final String CONTENT_DISPOSITION_INLINE = "inline";
     public static final String CONTENT_DISPOSITION_ATTACHMENT = "attachment";
+    public static final String CONTENT_DISPOSITION = "Content-Disposition";
```

Comment:
This change might be accidental?  I'm not against it though, since you seem to be alphabetizing things (but you forget to also move `CONTENT_DISPOSITION_FORMAT` up here).   So, you can keep this if you wish, but I noticed it's just moving this line to a different location